files/package.json

```diff
@@ -1,4 +1,5 @@
 {
+  "name": "<%= addonName %>",
```

Comment:
I don't think we want this to be the same as the addon -- some tools include the root package.json as part of the list of unique packages -- so... maybe this could be `"<%= addonName %>-root"`?

Reply:
Well, I was also wondering if this is valid, but at least yarn did not complain...

I actually spent an unreasonable amount of time on this rather small PR, debugging what `ember-cli-update` does. The thing is that this is what determines what `<addon-name>` is when ember-cli-update runs `ember new <addon-name> -b ...` as part of determining the diff for that update. So without it, it fails miserably. And with a name that does not match your actual addon name, you'll get a bunch of weird merge conflicts. The only thing that works is when the name matches.

Maybe there could be a different way to tell ecu what the addon name should be, e.g. in the json config? But that's not a thing atm AFAIK... /cc @kellyselden

Reply:
I just did a quick test with a monorepo, where the (private) root package has the same name as an actual workspace. Neither npm, yarn nor pnpm complained about it, and did their `install` job without problems.

> some tools include the root package.json as part of the list of unique packages

@NullVoxPopuli what other tools do you have in mind here, that could choke on this setup?

Reply:
Could be i misremembered, but trying turbo might be good

Reply:
I tried, with a very simple dummy pipeline. Running it with turbo worked, and I didn't see any warnings/errors related to the root package name... 🤷‍♂️

Off topic: turbo seems nice, haven't used it before. Should we use it here? Would enable us to drop things as `pretest`, right?

Reply:
> . Running it with turbo worked,
 
 most excellent.
 
 > Should we use it here? 
 
maybe :thinking: , it's a little awkward for small monorepos
 
 > Would enable us to drop things as pretest, right?
 
 yeah, exactly -- the main advantage I see is if you're running a task multiple times, the cache would skip the addon's build where as currently, we over build